charts/amazon-eks-pod-identity-webhook/templates/deployment.yaml

```diff
@@ -42,15 +42,15 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           command:
             - /webhook
-            - --annotation-prefix={{ .Values.config.annotationPrefix }}
-            - --aws-default-region={{ .Values.config.defaultAwsRegion }}
+            - --annotation-prefix={{ tpl .Values.config.annotationPrefix . }}
+            - --aws-default-region={{ tpl .Values.config.defaultAwsRegion . }}
             - --in-cluster=false
             - --logtostderr
             - --namespace={{ include "amazon-eks-pod-identity-webhook.namespace" . }}
             - --metrics-port={{ .Values.config.ports.metrics }}
             - --port={{ .Values.config.ports.webhook }}
-            - --sts-regional-endpoint={{ .Values.config.stsRegionalEndpoint }}
-            - --token-audience={{ .Values.config.tokenAudience }}
+            - --sts-regional-endpoint={{ tpl .Values.config.stsRegionalEndpoint . }}
```

Comment:
Looking at CI, stsRegionalEndpoint is a bool, but needs a string.

I guess `{{ print .Values.config.stsRegionalEndpoint }}` should help here.